ogcapi-stable/ogcapi-collections/src/main/java/de/ii/ogcapi/collections/domain/CollectionsConfiguration.java

```diff
@@ -27,7 +56,15 @@
     abstract class Builder extends ExtensionConfiguration.Builder {
 
     }
-
+  /**
+   * @en Add additional links to the Collections resource. The value is an array of
+   * link objects. Required properties of a link are a URI (href), a label (label)
+   * and a relation (rel).
+   * @de Erlaubt es, zusätzliche Links in der Ressource Feature Collections zu ergänzen.
+   * Der Wert ist ein Array von Link-Objekten. Anzugeben sind jeweils mindestens
+   * die URI (href), der anzuzeigende Text (label) und die Link-Relation (rel).
+   * @default []
+   */
   @JsonMerge(OptBoolean.FALSE)
     List<Link> getAdditionalLinks();
 
```

Comment:
@CabukEge Javadoc for `getCollectionIdAsParameter` and `getCollectionDefinitionsAreIdentical` are missing.